bin/rails

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env ruby
+begin
+  load File.expand_path("../spring", __FILE__)
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.